testing/integration/test/bin/load-zork.sh

```diff
@@ -47,14 +51,20 @@ if $RUNVNC; then
     x11vnc -display :10 -forever &
 fi
 
-if ! $PREBUILT; then
+if [[! $PREBUILT ]] || $NPM; then
```

Comment:
I don't think you need to change this line: it's still true that in all cases except `$PREBUILT` we'll need to populate `/test/src`.